src/Catty/ViewController/Stage/ScenePresenterViewController+Resources.swift

```diff
@@ -46,7 +46,7 @@ import CoreBluetooth
     }
 
     @nonobjc private func notifyUserAboutUnavailableResources(navigationController: UINavigationController) -> Bool {
-        let requiredResources = project.getRequiredResources()
+        let requiredResources = project.scene.getRequiredResources()
```

Comment:
The project might need some resources the Scene does not have, thus please call the `requiredResources` function of the project itself.